OpenSim/Simulation/Model/Model.cpp

```diff
@@ -1686,7 +1686,7 @@ void Model::writeMarkerFile(const string& aFileName)
  * @param aMarkerSet The new marker set to copy.
  * @return Number of markers that were successfully added to the model.
  */
-int Model::replaceMarkerSet(const SimTK::State& s, const MarkerSet& aMarkerSet)
+int Model::replaceMarkerSet(SimTK::State& s, const MarkerSet& aMarkerSet)
```

Comment:
> I'm happy to replace initSystem with any other call that keeps the State up to date with Property changes due to removing Markers and adding new ones.

editting should not invoke initSystem. This would be like a property editing invoking initSystem. Once the edits are complete and the Tool should invoke initSystem(). I see you had updated `Model::updateMarkerSet()` to invoke `initSystem()` and I should have caught that earlier.